ikasaneip/hospital/src/main/java/org/ikasan/hospital/dao/HospitalDao.java

```diff
@@ -76,9 +76,21 @@
 	 * @param flowName
 	 * @param startDate
 	 * @param endDate
+	 * @param size
 	 * @return
 	 */
-	public List<ExclusionEventAction> getActionedExclusions(List<String> moduleName, List<String> flowName, Date startDate, Date endDate);
+	public List<ExclusionEventAction> getActionedExclusions(List<String> moduleName, List<String> flowName, Date startDate, Date endDate, int size);
```

Comment:
Why are the HospitalDao changes done as part of this pull request. I was under the impression that this pull request was meant to bring new functionality related to replay service?